Ginger/GingerCoreNET/RunLib/Agent.cs

```diff
@@ -472,7 +472,30 @@ public void driverMessageEventHandler(object sender, DriverMessageEventArgs e)
                 OnPropertyChanged(Fields.Status);
             }
         }
+        public bool SupportVirtualAgent()
+        {
+            try
+            {
+
+                if (DriverClass == null)
+            {
+                DriverClass = RepositoryItemHelper.RepositoryItemFactory.GetDriverType(this);
+            }
+
+ 
+                if (DriverClass.GetInterfaces().Contains(typeof(IVirtualDriver)))
+                {
+                    return true;
+                }
+            }
+
+            catch(Exception e)
```

Comment:
Codacy found an issue: [Handle the exception or explain in a comment why it can be ignored.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6116697)